lms/services/exceptions.py

```diff
@@ -199,8 +190,8 @@ class CanvasFileNotFoundInCourse(Exception):
     error_code = "canvas_file_not_found_in_course"
 
     def __init__(self, file_id):
-        self.extra_details = {"file_id": file_id}
-        super().__init__(self.extra_details)
+        self.details = {"file_id": file_id}
+        super().__init__(self.details)
```

Comment:
There are some exception classes where `details` recently got erroneously renamed to `extra_details`: this `CanvasFileNotFoundInCourse` class, the `BlackboardFileNotFoundInCourse` below, and `CanvasGroupError` in another file. These all go to the [catch-all `api_error()` exception view](https://github.com/hypothesis/lms/blob/566cf7084773265314ebe46d51f17bd4510fa4e3/lms/views/api/exceptions.py#L159-L200) which calls [`self.error_response(..., details=context.extra_details)`](https://github.com/hypothesis/lms/blob/566cf7084773265314ebe46d51f17bd4510fa4e3/lms/views/api/exceptions.py#L191) and `error_response()` [uses the given details as the `details` dict in the error response](https://github.com/hypothesis/lms/blob/566cf7084773265314ebe46d51f17bd4510fa4e3/lms/views/api/exceptions.py#L223-L224). So what we're dealing with in this case is called `details`, not `extra_details`